### ISSUE TITLE
The refresh button appears almost transparent and can be easily missed when viewed in the default Editor theme of the IDE (Textmate).

### ISSUE THREAD

**Issue by R-Hannibal** [rstudio/rstudio#6670]
### System details

    RStudio Edition : Windows Desktop
    RStudio Version : Version 1.3.938
    OS Version      : Windows 10 Pro
    R Version       : 3.4.4

### Steps to reproduce the problem

1. Open the IDE with default Appearance options selected (Tools > Global Options > Appearance).
        a. If appearance options have already been changed, set 'Editor theme:' to 'Textmate (default)'.

### Describe the problem in detail
The refresh button blends almost seamlessly into the background when viewed in default themes of the IDE, making it very easy to miss for a first time or inexperienced user. 

Brightening the refresh symbol or providing a more distinct outline would make this button much easier to view.

![screenshot of low contrast icon in Viewer toolbar](https://user-images.githubusercontent.com/62304541/79368372-e407a080-7f14-11ea-8433-e349f93a23b3.PNG)

![screenshot of low contrast icons in multiple panes](https://user-images.githubusercontent.com/62304541/79368788-97709500-7f15-11ea-8642-fc858aaff1b9.png)

### Describe the behavior you expected
All buttons and options presented in the IDE are easy to distinguish and view.

### Notes
**[Note]** The refresh button appears on multiple panes of the IDE (Files, Plots, Packages, Viewer, etc.) and is displayed consistently in all of these panes (if we decide to enhance it in one place, it should be enhanced globally).

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
Yes, this is an issue with quite a few of the icons. These need to be made compliant with accessibility requirements, at least in TextMate (our default theme) and any future high-contrast themes we may create. https://www.w3.org/TR/WCAG21/#non-text-contrast

**Comment by gtritchie** [comment]
Another good example is the "clear console" broom at right of console toolbar:
<img width="127" alt="screenshot of console toolbar showing low contrast icon" src="https://user-images.githubusercontent.com/10569626/79369849-7bb1c280-7f06-11ea-9408-659f5a5bcda8.png">
